homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -97,6 +112,38 @@ class SensorType:
     ),
 }
 
+HUMIDIFIER_SENSOR_TYPES = {
```

Comment:
I think I would just make one SENSOR_TYPES dict for gateway and humidifier and all other devices, that will make it a lot easier in the future.